client/src/App.jsx

```diff
@@ -1,13 +1,86 @@
-import React from 'react';
+import React, { Component } from 'react';
 
 import MapView from './mapView';
 import Interface from './Interface';
+import fetchParklets from '../utils/fetchParklets';
+import fakeFilterFlags from '../utils/fakeFilterFlags';
 
-const App = () => (
-  <div id="home">
-    <MapView />
-    <Interface />
-  </div>
-);
+const PARKLETS_ENDPOINT = 'https://data.sfgov.org/resource/6a7x-cttf.json';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      activeParklets: [],
+      hiddenParklets: [],
+      error: '',
+      openFilter: false,
+      foodFilter: false,
+      wifiFilter: false,
+    };
+
+    this.handleFilters = this.handleFilters.bind(this);
+  }
+
+  componentDidMount() {
+    // IMPORTANT: Until db is implemented, fake data will be used to generate flags
+    // for a parklet that is open, has coffee, or wifi by RNG, this will change for
+    // every refresh of the page
+    fetchParklets(PARKLETS_ENDPOINT)
+      .then(fakeFilterFlags)
+      .then(data => this.refreshParkletDisplay(data))
+      .catch(err => this.setState({ error: err }));
+  }
+
+  refreshParkletDisplay(parklets) {
+    const { openFilter, foodFilter, wifiFilter } = this.state;
+    const hasFilter = openFilter || foodFilter || wifiFilter;
+    const activeParklets = [];
+    const hiddenParklets = [];
+
+    parklets.forEach((parklet) => {
```

Comment:
I would use reduce instead to sort parklets into an object with your two keys. Then you can do setState with `{...filteredObject}`.